iothub/service/samples/how to guides/MessagingClientSample/MessagingClientSample.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <LangVersion>9.0</LangVersion>
+
+    <!--
+    Warning for using pre-release nugets can be ignored since this sample 
+    can't target a main release yet.
+    -->
+    <NoWarn>NU5104</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="CommandLineParser" Version="2.9.1" />
+  </ItemGroup>
+
+  <!--<ItemGroup Condition="'$(Configuration)' == 'Release'">
```

Comment:
I see this same block in all the other samples, so I believe David has something in mind with this commented out code.

Reply:
We can put it in after we GA, so customers will get the shipped nuget tested if they run Release.

We can choose to only use project reference tho. If we add new functionality to an SDK and use it in a sample, it'd otherwise be broken in Release until after we release, so this may not have been my brightest idea.